models/iosxr/acl_interfaces/iosxr_acl_interfaces.yaml

```diff
@@ -0,0 +1,76 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: |
+    {
+        'metadata_version': '1.1',
+        'status': ['preview'],
+        'supported_by': 'network'
+    }
+
+NETWORK_OS: iosxr
+RESOURCE: acl_interfaces
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: iosxr_acl_interfaces
+  version_added: "2.10"
+  short_description: Manage adding and removing Access Control Lists (ACLs) from interfaces on devices running IOS-XR software.
```

Comment:
```suggestion
  short_description: Manage  Access Control Lists (ACLs) from  IOS-XR interfaces 
```

Reply:
This ends up being the page title in documentation so trying to trim it down a bit so it doesn't wrap on the page.